mne_bids_pipeline/_config.py

```diff
@@ -573,6 +573,18 @@
 Auto-detect "noisy" channels and mark them as bad.
 """
 
+
+find_bad_channels_extra_kws: dict[str, Any] = {}
+
+"""
+A dictionary of extra kwargs to pass to `mne.preprocessing.find_bad_channels_maxwell`
+. If kwargs are passed here that have dedicated config settings already, an error 
```

Comment:
If you click on the CI it should go [here](https://results.pre-commit.ci/run/github/167957354/1741024780.korkagCwQPWQdNh7Ez-1kw) where it says:
```
mne_bids_pipeline/_config.py:581:82: W291 Trailing whitespace
    |
579 | """
580 | A dictionary of extra kwargs to pass to `mne.preprocessing.find_bad_channels_maxwell`
581 | . If kwargs are passed here that have dedicated config settings already, an error 
    |                                                                                  ^ W291
582 | will be raised.
583 | For full documentation of the bad channel detection:
    |
    = help: Remove trailing whitespace
```
So:
```suggestion
. If kwargs are passed here that have dedicated config settings already, an error
```

Reply:
yup, I had fixed those, but for some reason the pushed version wasn't showing up here (and gave a 500 error on my branch).

`pre-commit run --all-files` is super helpful, I'll do that before I push from now on 🫡